js/weather-service.js

```diff
@@ -25,7 +28,7 @@
                 return null;
             }
             service.forcast.data.currently.day = moment.unix(service.forcast.data.currently.time).format('ddd')
-            service.forcast.data.currently.temperature = parseFloat(service.forcast.data.currently.temperature).toFixed(1);
+            service.forcast.data.currently.temperature = vsprintf(numberFormat, [parseFloat(service.forcast.data.currently.temperature).toFixed(1)]);
```

Comment:
The format is to add a whitespace if no "-" as First char.(only for Numbers with 4 chars)